src/main/java/gomedic/model/person/doctor/Department.java

```diff
@@ -0,0 +1,58 @@
+package gomedic.model.person.doctor;
+
+import static java.util.Objects.requireNonNull;
+
+import gomedic.commons.util.AppUtil;
+
+/**
+ * Represents the department of a Doctor.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDepartmentName(String)}
+ */
+public class Department {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Departments should only contain alphanumeric characters and spaces, and it should not be blank";
```

Comment:
Not sure whether we would allow number here, department 123 seems a bit weird but it doesn't violate anything too

Reply:
I think we could leave it as is for now in case of examples like "XRay Centre 2", shall we KIV?